pkg/twittermeow/client.go

```diff
@@ -417,4 +424,9 @@ func (c *Client) makeAPIRequest(ctx context.Context, apiRequestOpts apiRequestOp
 
 func (c *Client) SetActiveConversation(conversationID string) {
 	c.polling.SetActiveConversation(conversationID)
+	c.stream.startEventStream(conversationID)
```

Comment:
This also probably shouldn't be restarting the event stream every time. Just have one event stream active and change subscriptions when SetActiveConversation is called